src/v3/src/components/PhoneAuthenticator/PhoneAuthenticator.tsx

```diff
@@ -178,6 +177,10 @@ const PhoneAuthenticator: UISchemaElementComponent<UISchemaElementComponentWithV
             onChange={(e: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
               // Set new phone value without phone code
               setPhone(e.currentTarget.value);
+              // setTimeout needed because touched is being updated before validation
+              setTimeout(() => {
+                setTouched?.(true);
+              });
```

Comment:
This is a bit of a smell. @leemchale-okta @glenfannin-okta do you have any idea if we can avoid this?

Reply:
I felt the same but knew this would be a quick fix

@leemchale-okta can you try creating a new `useEffect` with the `phone` variable set as a dependency. Place this new useEffect _after_ the one that already exists, and inside this new useEffect execute the `setTouched` function with true and see if that works in the various scenarios.

Reply:
@glenfannin-okta i just tried it but it didnt work the field error comes up right away on page load

Reply:
Give it a try but I don't want to block us on this. If we decide to keep this code for now, please file a tech-debt JIRA for immediate followup

Reply:
I created this Draft PR, is this much better? It fixes the underlying issue and allows us to remove the settimeout

https://github.com/okta/okta-signin-widget/pull/2788